### PR TITLE
Typo Fixes

### PR DESCRIPTION
**Pull request by ParagEkbote**
# What does this PR do?

This PR aims to fix minor typos present in the notebooks. Closes #395, #427, #93, #126, #159, #128 .

Also, #51 and #458 can be closed since the suggested changes are out of scope.

## Who can review?

@stevhliu

### CODE REVIEW

File: diffusers_doc/en/write_own_pipeline.ipynb [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "\n",
```
Comment (stevhliu) [inline diffusers_doc/en/write_own_pipeline.ipynb:53]: <p>"The pipeline begins with random noise that has the same shape as the desired output, and then passes it through the model according to the number of steps set to run the denoising process for."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/530/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

File: examples/onnx-export.ipynb [hunk 2]
```diff
@@ -14,7 +14,7 @@
    "cell_type": "markdown",
```
Comment (stevhliu) [inline examples/onnx-export.ipynb:14]: <p>"It allows you to save your neural network's..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/530/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

**Comment by ParagEkbote** [comment]
I have updated the mentioned notebooks as per the advice given. Could you please review the changes?

cc: @stevhliu

**Review by stevhliu — APPROVED** [review]
Thanks for the fixes!

**Pull request merged**